bve/src/parse/mesh/instructions/tests/meshes/README.md

```diff
@@ -0,0 +1,19 @@
+# Test Meshes
+
+**This is not currently true, I currently can't normalize meshes, so results
```

Comment:
If it's not true, you should delete this from the README

Reply:
While I kinda agree, I don't really have a better place to put this information right now, and the former part is still useful.